src/cbor.js

```diff
@@ -72,11 +70,15 @@ exports.unmarshal = (input, opts) => {
       if (val instanceof cbor.Tagged) {
```

Comment:
Actually this can not happen here. We require `cbor` at the top and call `cbor.decode` and then check the decoded output against `cbor.Tagged` so this will always reference the same version of `cbor`.